keras_cv/models/object_detection/retina_net/retina_net.py

```diff
@@ -411,6 +417,19 @@ def test_step(self, data):
 
         return self.compute_metrics(x, {}, {}, sample_weight={})
 
+    def get_config(self):
+        return {
+            "classes": self.classes,
+            "bounding_box_format": self.bounding_box_format,
+            "backbone": self.backbone,
```

Comment:
don't we need to call `self.backbone.get_config` here?

Reply:
Since the backbone is a Keras concept, config can be loaded recursively (same thing we do in RetinaNetLabelEncoder with the AnchorGenerator).

But we were missing serialization registration -- Done